recipes/faiss/meta.yaml

```diff
@@ -0,0 +1,234 @@
+
+source:
+  url: https://github.com/facebookresearch/faiss/archive/v{{ version }}.tar.gz
+  sha256: e1a41c159f0b896975fbb133e0240a233af5c9286c09a28fde6aefff5336e542
+  patches:
+    - patches/0001-change-python-Makefile-to-point-to-libfaiss.-SHLIB_E.patch
+    - patches/0002-remove-fPIC-flag-for-msvc-compat.patch  # [win]
+
+build:
+  number: {{ number }}
+  # TODO: figure out compiler errors on windows
+  skip: true                      # [win]
+  # TODO: remove this to retry building GPU on the feedstock
+  skip: true                      # [cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+outputs:
+  - name: faiss-proc
+    version: 1.0.0
+    build:
+      number: 0
+      string: {{ faiss_proc_type }}
+    test:
+      commands:
+        - exit 0
+    about:
+      home: https://github.com/conda-forge/faiss-feedstock
+      license: MIT
+      license_family: MIT
+      summary: A meta-package to select CPU or GPU build for faiss.
+
+  - name: libfaiss
+    script: build-lib.sh          # [not win]
+    script: build-lib.bat         # [win]
+    build:
+      number: {{ number }}
+      string: "h{{ PKG_HASH }}_{{ number }}_{{ faiss_proc_type }}"
+      run_exports:
+        - faiss-proc * {{ faiss_proc_type }}
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+        - cmake
+        - libgomp                 # [linux]
+        - llvm-openmp             # [osx or win]
+        - autotools_clang_conda   # [win]
+      host:
+        - libblas
+        - liblapack
+      run:
+        - libblas
+        - liblapack
+      run_constrained:
+        - faiss-proc * {{ faiss_proc_type }}
+
+    test:
+      commands:
+        # shared
+        - test -f $PREFIX/lib/libfaiss.so                           # [linux]
+        - test -f $PREFIX/lib/libfaiss.dylib                        # [osx]
+        - if not exist %PREFIX%\\Library\\bin\\libfaiss.dll exit 1  # [win]
+
+        # absence of static libraries
+        - test ! -f $PREFIX/lib/libfaiss.a                          # [not win]
+        - if exist %PREFIX%\\Library\\lib\\libfaiss.lib exit 1      # [win]
+
+        # conda tools
+        - conda inspect linkages -p $PREFIX $PKG_NAME               # [not win]
+        - conda inspect objects -p $PREFIX $PKG_NAME                # [osx]
+
+        # headers
+        {% for each_header in headers %}
+        - test -f $PREFIX/include/faiss/{{ each_header }}                                    # [unix]
+        - if not exist %LIBRARY_INC%\\faiss\\{{ "\\".join(each_header.split("/")) }} exit 1  # [win]
+        {% endfor %}
+
+  - name: faiss
+    script: build-pkg.sh          # [not win]
+    script: build-pkg.bat         # [win]
+    build:
+      number: {{ number }}
+      string: "py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ number }}_{{ faiss_proc_type }}"
+      run_exports:
+        - faiss-proc * {{ faiss_proc_type }}
```

Comment:
Can you remove this?

Reply:
The whole of `run_exports`? Sure I can, I just thought it would be an added safety.

Reply:
Yes. A python package generally doesn't need `run_exports` like this.